glean-core/android/src/main/java/mozilla/telemetry/glean/acmigration/GleanACDataMigrator.kt

```diff
@@ -38,13 +44,19 @@ internal class GleanACDataMigrator(
      *        the remaining metadata is not loaded and will be `null`.
      * @param sequenceNumbers the mapping between AC storage names and their sequence
      *        numbers.
-     * @return a [Triple] containing the keys and the values representing the sequence
-     *         numbers map, with the addition of the number of elements in the map.
+     * @param metricsPingLastSentDate the last time the metrics ping was sent (if ever).
      */
     internal data class ACMetadata(
         val alreadyMigrated: Boolean,
-        val sequenceNumbers: Map<String, Int>
+        val sequenceNumbers: Map<String, Int>,
+        val metricsPingLastSentDate: String?
     ) {
+        /**
+         * Get a [Triple] containing data to be passed to the FFI layer.
+         *
+         * @return a [Triple] containing the keys and the values representing the sequence
+         *         numbers map, with the addition of the number of elements in the map.
+         */
         fun toFfi(): Triple<StringArray?, IntArray?, Int> {
```

Comment:
With now another thing stored in this class, couldn't we just create and store the Ffi data in the constructor and not bother with another function call and an opaque triple?

Reply:
This is the last bit of data going there, _I promise_! Also note that `toFfi` is only called if we really need to migrate in `Glean.kt`, so I'd rather leave this as it is.